### ISSUE TITLE
New internal dependencies upgrade will break the app for not label type for height.

### ISSUE THREAD

**Issue by kieykouch** [Popmotion/popmotion#421]
## Add the package name to bug title
style-value-types: App will break if height is not labeled with vh or px
error message: prop.match is not a function

https://github.com/Popmotion/popmotion/blob/ad8e7464f6553ed8b18d159fe924176408510f9d/packages/style-value-types/src/value-types/complex.ts
Line 46

## Describe the bug
We are using react-pose to create animation. After this dependency updated, the app won't run. 
We found out that we have to label height with px or vh, otherwise, we will see the above error message, prop.match is not a function.

### Error code
```
import posed from "react-pose";
import { tween } from "popmotion";

const AnimatedHeader = posed.div({
  hidden: {
    height: 0
  },
  visible: {
    height: '100',
    transition: (props) => tween({ ...props, duration: 1500 })
  }
});
```

## Additional context
I just want to let the you know or help out other people who might run into this issue. Plus, I should have labeled px as well.

c.c @InventingWithMonster 
You can close this issue, cheers.

**Comment by InventingWithMonster** [comment]
@kieykouch Hey there

I'm gonna look into this because we should probably handle this better, but if you're not defining a value unit, height should be a `number` type, whereas in your example it's a string

**Comment by InventingWithMonster** [comment]
Just released `style-value-types@3.0.5`, and your original functionality should be restored.

**Comment by clementizard** [comment]
Hello!

Sorry for digging up this issue, but this one appears to happens not only on height but also on font size, and many other things (that i haven't found out yet, i still have this issue)

The strange thing is that it continues to work, but a lot slower.

Do you have any list of properties that can't be set without "px" at the end of the string?
Also, i use framer-motion: 2.6.3
